src/.clang-tidy

```diff
@@ -7,6 +7,7 @@ modernize-use-default-member-init,
 modernize-use-nullptr,
 performance-faster-string-find,
 performance-for-range-copy,
+performance-inefficient-string-concatenation,
```

Comment:
Not sure. This makes the code pretty verbose and it is unclear whether append or strprintf should be used. Also, while this check is disabled for non-loops. In our code it is only triggered on errors in loops.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/26642#issuecomment-1341293638).